packages/legend-application-query/src/components/QueryEditor.tsx

```diff
@@ -819,6 +819,114 @@ const QueryEditorHeaderContent = observer(
   },
 );
 
+const QueryEditorExistingQueryVersionRevertModal = observer(
```

Comment:
Just from a style/code organization perspective, could we move this modal component into its own file? I've noticed a lot of our files contain multiple components within them and can get really long. I think it would be cleaner to keep each file as small as possible and make sure each component (at least components that are relatively large) get their own file. Thoughts?

Reply:
good point. 
Moved it to its own file